mmv1/third_party/terraform/tests/resource_compute_shared_vpc_test.go

```diff
@@ -47,7 +63,7 @@ func TestAccComputeSharedVpc_basic(t *testing.T) {
 					testAccCheckComputeSharedVpcHostProject(t, hostProject, false),
 					testAccCheckComputeSharedVpcServiceProject(t, hostProject, serviceProject, false),
 				),
-			},
+			}
```

Comment:
Note: tests are failing due to provider compilation failing:

```
./google/resource_compute_shared_vpc_test.go:66:5: missing ',' before newline in composite literal
```

Please [generate the provider](https://googlecloudplatform.github.io/magic-modules/docs/getting-started/generate-providers/) and [run unit tests](https://googlecloudplatform.github.io/magic-modules/docs/getting-started/run-provider-tests/#run-unit-tests) even if you're not able to run acceptance tests to make sure this is fixed.